### ISSUE TITLE
Screen readers don't read movement hint

### ISSUE THREAD

**Issue by warm-coolguy** [Dataport/polar#672]
**Describe the bug**
On using NVDA or the likes, the zoom instructions when scrolling across the map are not read. There may be some users that can visually use the map, but can't read the instructions. For those, the instructions should be read. We need to find a way to do this politely and maybe with a timeout so that scrolling around the page doesn't spam screen readers.

**Expected behavior**
NVDA reads the input.
